android/app/src/main/res/values-ro/strings.xml

```diff
@@ -517,7 +517,7 @@
 	<string name="error_enter_correct_twitter_page">Introdu o adresă web sau un nume de utilizator Twitter valabil</string>
 	<string name="error_enter_correct_vk_page">Introdu o adresă web sau un nume de cont VK valabil</string>
 	<string name="error_enter_correct_line_page">Introdu o adresă web LINE valabilă sau un ID LINE</string>
-	<string name="placepage_add_place_button">Adaugă un loc pe hartă</string>
+	<string name="placepage_add_place_button">Adăugați un loc pe OpenStreetMap</string>
```

Comment:
`Adăugați` is more formal, other messages are less formal, this should probably revert to `Adaugă`?